wom/src/main/scala/wom/graph/CallNode.scala

```diff
@@ -43,17 +43,26 @@ final case class WorkflowCallNode private(override val name: String,
 
 object TaskCall {
   def graphFromDefinition(taskDefinition: TaskDefinition): ErrorOr[Graph] = {
-    def linkOutput(call: GraphNode)(output: OutputDefinition): ErrorOr[GraphNode] = call.outputByName(output.name).map(out => PortBasedGraphOutputNode(output.name, output.womType, out))
+    val taskDefinitionLocalName = LocalName(taskDefinition.name)
+    
+    // Creates an identifier for an input or an output
+    // The localName is the name of the input or output
+    // The FQN combines the name of the task to the name of the input or output
+    def identifier(name: LocalName) = WomIdentifier(name, taskDefinitionLocalName.combineToFullyQualifiedName(name))
```

Comment:
so this identifier isn't really *fully* qualified, it's just qualified to the local name of the task definition?

Reply:
yes, although this method creates a CallNode and a Graph purely from a TaskDefinition. So the name of this task definition + name of the output is as much as we're going to get here which could be another nome for "fully qualified" 😄